db/migrate/20160825213523_change_load_balancer_listener_port_to_port_range.rb

```diff
@@ -0,0 +1,55 @@
+class ChangeLoadBalancerListenerPortToPortRange < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        # convert load_balancer_port and instance_port into ranges, preserving
+        # existing values
+        execute <<-SQL
+          ALTER TABLE load_balancer_listeners
+            ALTER COLUMN load_balancer_port SET DATA TYPE int4range
```

Comment:
@selmanj  I am sorry for that I used word 'arel', yes you are right I meant ActiveRecord